modules/monitoring-resources-reference-for-the-cluster-monitoring-operator.adoc

```diff
@@ -16,16 +16,13 @@ Use this information when you want to configure API endpoint connections to retr
 
 [IMPORTANT]
 ====
-In certain situations, accessing API endpoints can degrade the performance and scalability of your cluster, especially if you use endpoints to retrieve, send, or query large amounts of metrics data.
+In certain situations, accessing endpoints can degrade the performance and scalability of your cluster, especially if you use endpoints to retrieve, send, or query large amounts of metrics data.
 
 To avoid these issues, follow these recommendations:
 
 * Avoid querying endpoints frequently. Limit queries to a maximum of one every 30 seconds.
-* Do not try to retrieve all metrics data via the `/federate` endpoint for Prometheus.
-Query it only when you want to retrieve a limited, aggregated data set.
-For example, retrieving fewer than 1,000 samples for each request helps minimize the risk of performance degradation.
+* Do not try to retrieve all metrics data via the /federate endpoint. Query it only when you want to retrieve a limited, aggregated data set. For example, retrieving fewer than 1,000 samples for each request helps minimize the risk of performance degradation.
```

Comment:
shall we change /federate to ` /federate`
```suggestion
* Do not try to retrieve all metrics data via the ` /federate` endpoint. Query it only when you want to retrieve a limited, aggregated data set. For example, retrieving fewer than 1,000 samples for each request helps minimize the risk of performance degradation.
```
to consistent with https://86976--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/monitoring/accessing-third-party-monitoring-apis.html#resources-reference-for-the-cluster-monitoring-operator_accessing-monitoring-apis-by-using-the-cli